alacritty/src/display.rs

```diff
@@ -170,17 +171,26 @@ impl Display {
         // Guess the target window dimensions.
         let metrics = GlyphCache::static_metrics(config.ui_config.font.clone(), estimated_dpr)?;
         let (cell_width, cell_height) = compute_cell_size(config, &metrics);
-
-        let dimensions = GlyphCache::calculate_dimensions(
-            &config.ui_config.window,
-            estimated_dpr,
-            cell_width,
-            cell_height,
-        );
+        let (padding_x, padding_y) = compute_padding_size(config, estimated_dpr);
```

Comment:
What do we need the padding for if we do not have a window size set?

Reply:
We always calculate the min size, and that takes into account the padding.

Reply:
Why?

Reply:
We always want a min size to be set, as we never want to have a window that is too small.